docs/Commands.md

```diff
@@ -25,3 +25,7 @@ When the [Server Cache Path](Preferences.md#server-cache-path) is activated, the
 This command re-triggers the [XML Validation](Validation.md#xml-validation) for the all opened XML files.
 
 When the [Server Cache Path](Preferences.md#server-cache-path) is activated, the command clears the remote grammar cache and revalidates all opened files.
+
+## Restart XML Language Server connection
+
+This command triggers `XML: Restart Language Server`, which restarts the connection from the vscode client and the XML language server.

```

Comment:
Does it just restart the connection, or does LemMinX restart?

Reply:
Just the connection, it calls LanguageClient.start() and LanguageClient.stop()

Reply:
In hind sight, using the name `XML: Restart Language Server Connection` might have been better

Reply:
I think that it actually restarts the server. When I run `watch jps`, then issue the command, the PID of the server changes. I think it might be a better idea to say "restart the server", in the documents

Reply:
I would keep the description simple :

This command restarts the XML language server.

I don't think we need to go into any detail on how it does it (stop/start) or to specify that we do it vita the vscode language client API. As far as the user is concerned. the current language server process is shut down, and a new one replaces it.